packages/web/src/app/doc-pages/components/modal-doc/modal-ceg/modal-ceg.component.html

```diff
@@ -5,7 +5,7 @@
     <div>Er du sikker på at du vil du godkjenne at Google bruker data om deg?</div>
   </div>
   <div slot="illustration">
-    <img alt="" src="/assets/modal/Empty state.png" />
+    <elvia-illustrations-broken />
   </div>
```

Comment:
Will the layout remain correct if we put the slot directly on the `elvia-illustrations` element?

Reply:
Maybe, but it is probably safer to just do it like this. And I don't think the CEG code generator will properly support it out-of-the-box